app/views/spree/admin/products/_form.html.haml

```diff
@@ -0,0 +1,106 @@
+%div{"data-hook" => "admin_product_form_fields"}
```

Comment:
data-hook should be removed right?

Reply:
This data-hook is used for CSS : https://github.com/openfoodfoundation/spree/blob/41906362d931695e0616194341a68d2c4c85aaaf/backend/app/assets/stylesheets/admin/sections/_products.scss#L10

:/